app/views/katello/layouts/foreman_with_bastion.html.erb

```diff
@@ -2,4 +2,12 @@
   <%= render :partial => 'layouts/application_content' %>
 <% end %>
 
+<% content_for(:stylesheets) do %>
+  <%= stylesheet_link_tag *webpack_asset_paths('katello', :extension => 'css'), "data-turbolinks-track" => true %>
+<% end %>
+
+<% content_for(:javascripts) do %>
+  <%= javascript_include_tag *webpack_asset_paths('katello', :extension => 'js'), "data-turbolinks-track" => true %>
```

Comment:
@danseethaler the defer attribute would be placed here:
```ruby
javascript_include_tag *webpack_asset_paths('katello', :extension => 'js'), "data-turbolinks-track" => true, 'defer' => 'defer'
```

Reply:
I don't think this is required, if you expose your components (like core does it at https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/componentRegistry.js ) then you can simply use https://github.com/theforeman/foreman/blob/develop/app/helpers/reactjs_helper.rb#L2 from your rails views.

Reply:
Correct, but Katello doesn't have many pages that would use rails views. It's mostly angular. So I think that you either want to add react component to existing angular views and you can't use that helper or you're (re)writing a completely new page in react and then there's no point for adding a rails view.

Reply:
regardless, you should use the mounter service imho.